### ISSUE TITLE
Today widget

### ISSUE THREAD

**Issue by arzzcom** [moimz/iCoinTicker#9]
Like this 

<img width="333" alt="2017-07-12 17 43 22" src="https://user-images.githubusercontent.com/4408201/28109303-9d5696ba-6729-11e7-8696-62be67156920.png">

**Comment by remyx11** [comment]
Looks cool! Do you think version 4 will be downloadable via GitHub again?

**Comment by arzzcom** [comment]
@remyx11 Ofcourse! version 3.0 also available via GitHub soon. I am trying!
But I know, Today widget is allowed sandboxing app. maybe....
So, v4.0.0 via GitHub isn't include Today widget.

**Comment by arzzcom** [comment]
@remyx11 Could you test v3.2.0 app bundle with 3rd party developer signin.
[iCoinTicker.zip](https://github.com/moimz/iCoinTicker/files/1141862/iCoinTicker.zip)

If this app bundle launch or not in your mac, please tell me. thanks!

**Comment by remyx11** [comment]
Nope, it won't open. It flashes like its about to open, but thats it.

**Comment by arzzcom** [comment]
[iCoinTicker.zip](https://github.com/moimz/iCoinTicker/files/1141931/iCoinTicker.zip)

@remyx11 How about this?

**Comment by remyx11** [comment]
Yes, works!

**Comment by arzzcom** [comment]
@remyx11 Thanks for testing. 
I released v3.2.0 via GitHub. :)
